### PR TITLE
feat: Withdraw funds to a crypto address

### PR DESCRIPTION
**Pull request by piotr-roslaniec**

### CODE REVIEW

File: src/withdraw/WithdrawAPI.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   * @see https://docs.pro.coinbase.com/#crypto
+   */
+  async postCryptoWithdrawal(
+    amount: number,
+    currency: string,
+    cryptoAddress: string,
+    destinationTag?: string
+  ): Promise<CryptoWithdrawal> {
+    const resource = WithdrawAPI.URL.WITHDRAWALS.CRYPTO;
+    const withdrawal: any = {amount, crypto_address: cryptoAddress, currency};
```
Comment (bennyn) [inline src/withdraw/WithdrawAPI.ts:34]: Cool things going on here! 🤘 Can you type the `withdrawal` const?

**Comment by bennyn** [comment]
@piotr-roslaniec thank you for your contribution! I am really happy about this. Is your pull request ready for review?

**Comment by piotr-roslaniec** [comment]
@bennyn Sure, please go ahead with the review.

**Comment by bennyn** [comment]
Great. It's released with v2.3.0. Thank you for your contribution! 🏆

**Pull request merged**